README.md

```
@@ -20,6 +20,39 @@ Run the server:
 bundle exec jekyll serve
 ```
 
+## Career Changelog
+
+`_includes/career/changelog.html` is the source-of-truth narrative for my career experience, organized per role using the Keep-a-Changelog format. `resume.json:work[]` carries the compact, recruiter-facing distillation that the [Resume](#resume) pipeline converts into `resume.yml` and `resume-full.html`.
+
+The intended editing flow is changelog-first: changes land in `_includes/career/changelog.html`, then the matching `resume.json:work[]` entry gets a synced summary and highlights per the "summary is a compact distillation of self-sufficient highlights" rule.
+
+To enrich one role at a time, invoke the project-local skill:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)